helm/resource_release_test.go

```diff
@@ -346,10 +366,10 @@ func TestAccResourceRelease_updateAfterFail(t *testing.T) {
 			ExpectError:        regexp.MustCompile("invalid resource name"),
 			ExpectNonEmptyPlan: true,
 		}, {
-			Config: testAccHelmReleaseConfigBasic(testResourceName, namespace, name, "7.1.0"),
+			Config: fixed,
```

Comment:
This was actually (successfully) testing the wrong thing. Here we want to make a release for a chart that would fail, then install the same chart again successfully.